.github/workflows/node-chrome-test.yml

```diff
@@ -0,0 +1,37 @@
+name: Main Test Flow
+
+on:
+  push:
+    branches:
+      - '*'
+      - '*/*'
+      - '!master'
```

Comment:
You should probably include the master branch to be in safe side. Otherwise the merged state of some PRs will go untested.

Reply:
Well, that makes the config line simpler.